docs/wiki/guides/current/Position-Hold-v4-6.md

```diff
@@ -0,0 +1,149 @@
+# Position Hold 4.6
+
+Betaflight 4.6 introduces Altitude Hold and Position Hold
+
+When used together, these features allow the pilot to hit a switch and the aircraft will hover at a fixed location and constant altitude, in Angle Mode. The sticks can then be used to ajust altitude and location. A GPS is essential, and a Magnetometer is strongly recommended.
+
+## Quick-Start
+
+- Include `ALTITUDE_HOLD` and `POSITION_HOLD` in your firmware build.
+- Ensure your GPS is working properly
+- Ideally, include a Magnetometer and verify that it is working properly
+- Set the hover throttle value accurately
+- Enable both Altitude Hold and Position Hold on the same Mode switch
+- Hover the quad and test carefully; the aircraft will be in Angle mode and stick responses will be different from usual.
+
+## Altitude Hold
+
+Altitude Hold can be installed and tested independently of Position Hold.
+
+Altitude Hold is intended to keep the quad at a constant altitude, automatically adjusting the throttle as required. The throttle must either be set to zero, or put somewhere within a deadband either side of the normal hover point, for the altitude to be kept stable. The target altitude can be adjusted up or down by putting the throttle above, or below, the deadband.
```

Comment:
'either be set to zero' - this is so counterintuitive

Reply:
It's how the code currently works - and the guide should explain how it works.
Most pilots get accustomed to this behaviour fairly quickly.
It's definitely nice to be able to quickly jam throttle to zero and be sure the quad won't climb or fall.  Otherwise if  the throttle deadband is configured to be fairly narrow, and the pilot in a hurry to hold altitude, they can accidentally find the quad climbing or descending because throttle was accidentally outside deadband.
Being able to just jam throttle to zero and have it reliably still is actually very handy, and worth the inconsistency.